### PR TITLE
feat: add shopify logo

### PR DESCRIPTION
**Pull request by cbernat**
Added shopify logo, and updated library to fix header. 

image

http://cdn.fromdoppler.com/doppler-webapp/build984/#/integrations/shopify

### CODE REVIEW

File: src/components/Integrations/Shopify/Shopify.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ import React, { useState, useEffect } from 'react';
 import { Helmet } from 'react-helmet';
 import Loading from '../../Loading/Loading';
 import { InjectAppServices } from '../../../services/pure-di';
+import logo from './logo.svg';
+import './Shopify.css';
```
Comment (andresmoschini) [inline src/components/Integrations/Shopify/Shopify.js]: What do you think about using lowercase by default for the file names?

Comment (andresmoschini) [reply src/components/Integrations/Shopify/Shopify.js]: (if you change it, remember to also change the file name)

File: src/components/Integrations/Shopify/Shopify.css [hunk 2]
```diff
@@ -0,0 +1,8 @@
+img {
+    width: 10%;
```
Comment (andresmoschini) [inline src/components/Integrations/Shopify/Shopify.css]: Why is it a percentaje? is it fine apply this style for all `img`s?

File: src/components/Integrations/Shopify/Shopify.js [hunk 3]
```diff
@@ -73,7 +75,7 @@ const Shopify = ({ dependencies: { shopifyClient } }) => {
                 <div className="block dp-integration__status">
                   <div className="status__info">
                     <div>
-                      {/*<img src="img/integration__logo.png" alt="Shopify logo"/> this should be added as a class, and will be added later*/}
+                      <img src={logo} alt="Shopify logo" />
```
Comment (IgnacioRodrigues) [inline src/components/Integrations/Shopify/Shopify.js]: What do you think about using class for show logo the Shopify or integrations? As Doppler logo? 
`<h1 class="logo-doppler-new"><a target="_blank" href="https://www.fromdoppler.com/?utm_source=app&amp;utm_medium=landing&amp;utm_campaign=login" rel="noopener noreferrer">Doppler</a></h1>`

File: src/components/Integrations/Shopify/Shopify.js (hunk 1 above)
Comment (cbernat) [reply src/components/Integrations/Shopify/Shopify.js]: For all? We were using uppercase for components...

File: src/components/Integrations/Shopify/Shopify.css (hunk 2 above)
Comment (cbernat) [reply src/components/Integrations/Shopify/Shopify.css]: All images elements inside shopify component only. But I better add a class..

File: src/components/Integrations/Shopify/Shopify.js (hunk 1 above)
Comment (andresmoschini) [reply src/components/Integrations/Shopify/Shopify.js]: Oh, I did not understand the logic, it has the same name than the component. I understand it now. Thanks.

File: src/components/Integrations/Shopify/Shopify.css (hunk 2 above)
Comment (andresmoschini) [inline src/components/Integrations/Shopify/Shopify.css]: 10% regarding what?

Comment (cbernat) [reply src/components/Integrations/Shopify/Shopify.css]: The idea of having css inside a component is that it applies only to this component.

Comment (cbernat) [reply src/components/Integrations/Shopify/Shopify.css]: The outer parent container.
image mmm doesn't seem to be a good idea. I will change it.

**Pull request merged**